satpy/tests/enhancement_tests/helpers.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2017-2025 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Helpers for enhancement testing."""
```

Comment:
Codecov is complaining about almost every line in this module not being checked during tests. Are you sure these are being used?

Reply:
Oh is that because "utils" is being used instead? Should this be removed?